conf/settings.py

```diff
@@ -34,6 +34,8 @@
 
 # Semantic versioning + build version
 VERSION = f"0.0.0_{BUILD_VERSION}"
+GDAL_LIBRARY_PATH = "/opt/homebrew/opt/gdal/lib/libgdal.dylib"
+GEOS_LIBRARY_PATH = "/opt/homebrew/opt/geos/lib/libgeos_c.dylib"
```

Comment:
Definitely should not be here, `/opt/homebrew` is Mac specific and wont work inside any linux/windows environments.

```suggestion
GDAL_LIBRARY_PATH = os.getenv("GDAL_LIBRARY_PATH")
GEOS_LIBRARY_PATH = os.getenv("GEOS_LIBRARY_PATH")
```

For local development set these values in OS/Pycharm run configuration:
```sh
export GDAL_LIBRARY_PATH="/opt/homebrew/opt/gdal/lib/libgdal.dylib"
export GEOS_LIBRARY_PATH="/opt/homebrew/opt/geos/lib/libgeos_c.dylib"
```

Reply:
Changed, thanks